### ISSUE TITLE
Password Reset Functionality Broken

### ISSUE THREAD

**Issue by mmedic** [sct/overseerr#2864]
### Description

When I request a password reset, the link in format `https://server.com/resetpassword/8c9a5e0d-5c9c-426e-a9a5-c56fd8b6af4e` gets emailed to the user. Once I put in the new password and click the button, request gets submitted but response is 400
```
{"message":"request.params.guid should be number","errors":[{"path":".params.guid","message":"should be number","errorCode":"type.openapi.validation"}]}
```

I think the issue is that OpenAPI spec says this should be a number and it is being enforced: https://github.com/sct/overseerr/blob/develop/overseerr-api.yml#L3173

### Version

1.29.1

### Steps to Reproduce

1. Request password reset
2. Click on link in email (link is in format: `https://server.com/resetpassword/8c9a5e0d-5c9c-426e-a9a5-c56fd8b6af4e`
3. Overseerr responds with 400 expecting a number instead of a GUID

### Platform

desktop

### Device

Macbook Pro

### Operating System

MacOS

### Browser

Chrome

### Code of Conduct

- [X] I agree to follow Overseerr's Code of Conduct

**Comment by TheCatLady** [comment]
@danshilm, looks like this bug may have been introduced in #2478?

**Comment by danshilm** [comment]
Fixed in [#2834](https://github.com/sct/overseerr/pull/2834)

**Comment by mmedic** [comment]
Do you know when this is going be released?
